specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2018-01-01-preview/schemaregistry.json

```diff
@@ -29,104 +28,263 @@
+                "description": "URL location of schema, identified by schema ID."
+              },
+              "X-Schema-Version": {
+                "type": "integer",
+                "description": "Version of the returned schema."
+              }
+            },
             "schema": {
-              "$ref": "#/definitions/Result"
+              "$ref": "#/definitions/SchemaId"
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "An error response received from the Azure Schema Registry service.",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/definitions/ServiceErrorResponse"
             }
           }
         }
       }
     }
   },
   "definitions": {
-    "Result": {
-      "description": "Sample result definition",
+    "SchemaId": {
+      "type": "object",
+      "description": "JSON Object received from the registry containing schema identifiers.",
       "properties": {
-        "sampleProperty": {
+        "id": {
           "type": "string",
-          "description": "Sample property of type string"
-        }
-      }
-    },
-    "ErrorResponse": {
-      "description": "Error response.",
-      "properties": {
-        "error": {
-          "$ref": "#/definitions/ErrorDefinition",
-          "description": "The error details."
+          "description": "Schema ID that uniquely identifies a schema in the registry namespace."
         }
       }
     },
-    "ErrorDefinition": {
-      "description": "Error definition.",
-      "properties": {
-        "code": {
-          "description": "Service specific error code which serves as the substatus for the HTTP error code.",
-          "type": "string",
-          "readOnly": true
-        },
-        "message": {
-          "description": "Description of the error.",
-          "type": "string",
-          "readOnly": true
-        },
-        "details": {
-          "description": "Internal error details.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ErrorDefinition"
-          },
-          "readOnly": true
-        }
-      }
+    "ServiceErrorResponse": {
+      "description": "Error response returned from Azure Schema Registry service.",
+      "type": "string"
     }
   },
   "parameters": {
-    "SubscriptionIdParameter": {
-      "name": "subscriptionId",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "description": "The Azure subscription ID. This is a GUID-formatted string (e.g. 00000000-0000-0000-0000-000000000000)"
-    },
-    "ResourceGroupNameParameter": {
-      "name": "resourceGroupName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "description": "The name of the resource group.",
-      "x-ms-parameter-location": "method"
-    },
     "ApiVersionParameter": {
```

Comment:
Your operations need to reference this parameter (it won't be an actual API parameter though, the code generators do the right thing here).